test/test_file.py

```diff
@@ -0,0 +1,45 @@
+# This file represents all known cases of special-case indentations
```

Comment:
Yep, found it invaluable for testing. But now that I've had the CI tests fail I see that there's sort of one in the `specs` folder. I'll have to look into that more (also to see why the tests are failing...)